x-pack/plugins/enterprise_search/server/plugin.ts

```diff
@@ -46,11 +56,62 @@ export class EnterpriseSearchPlugin implements Plugin {
   }
 
   public async setup(
-    { http, savedObjects, getStartServices }: CoreSetup,
-    { usageCollection }: PluginsSetup
+    { capabilities, http, savedObjects, getStartServices }: CoreSetup,
+    { usageCollection, security, features }: PluginsSetup
   ) {
-    const router = http.createRouter();
     const config = await this.config.pipe(first()).toPromise();
+
+    /**
+     * Register space/feature control
+     */
+    features.registerFeature({
+      id: 'enterprise_search',
+      name: 'Enterprise Search',
+      order: 0,
+      icon: 'logoEnterpriseSearch',
+      navLinkId: 'app_search', // TODO: We need to get this working with multiple plugins :(
```

Comment:
Just want to flag this as something that's not ideal but will work until it gets looked at/fixed Kibana-side.

Without this navLinkId, the space doesn't properly hide App Search from the nav bar unfortunately.

Reply:
Once https://github.com/elastic/kibana/pull/70538 is merged in, I can change `navLinkId` to something like `'_'` since it will no longer be needed 🎉